src/ad-services/krux/index.js

```diff
@@ -82,8 +82,8 @@ class Krux {
 	 * @returns {void}
 	 */
 	importUserData() {
-		const user = getKruxData('kxuser');
-		const segments = getKruxData('kxsegs');
+		const user = getKruxData('kxuser') || getKruxData('kxwikia_user');
```

Comment:
Just wonder if we shouldn't make it configurable in context (e.g. `services.krux.userKey`) and use it as a fallback 
```suggestion
		const user = getKruxData('kxuser') || getKruxData(context.get('services.krux.userKey'));
```